internal/server/webhooks.go

```diff
@@ -31,9 +29,11 @@ func RequireHeaders(headers []kobold.Header, handler http.Handler) http.Handler
 func NewPushWebhook(id string, subs []chan events.PushData, ph events.PayloadHandler) http.Handler {
 	logger := log.With().Str("endpoint", id).Logger()
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		logger.Info().Str("endpoint", id).Msg("push event received")
+		var ct string = r.Header.Get("Content-Type")
+		logger.Info().Str("endpoint", id).Msg(fmt.Sprintf("push event received with Content-Type: %s", ct))
```

Comment:
[zerolog](https://github.com/rs/zerolog) is based on structured data. So it would be more idiomatic to add another `Str` call before the message

Something like below:

```go
logger.Info().Str("endpoint", id).Str("contentType", ct).Msg("push event received")
```